finality-aleph/src/new_network/data_network/rmc_network.rs

```diff
@@ -0,0 +1,33 @@
+use crate::{
+    aggregator::SignableHash,
+    crypto::Signature,
+    new_network::data_network::{split::DataNetwork, Recipient},
+    Error, NodeIndex,
+};
+use aleph_bft::SignatureSet;
+use sp_api::BlockT;
+
+pub(crate) type RmcNetworkData<B> =
+    aleph_bft::rmc::Message<SignableHash<<B as BlockT>::Hash>, Signature, SignatureSet<Signature>>;
+
+pub(crate) struct RmcNetwork<B: BlockT> {
+    inner: DataNetwork<RmcNetworkData<B>>,
+}
+
+impl<B: BlockT> RmcNetwork<B> {
+    pub(crate) fn new(inner: DataNetwork<RmcNetworkData<B>>) -> Self {
+        RmcNetwork { inner }
+    }
+
+    pub(crate) fn send(
+        &self,
+        data: RmcNetworkData<B>,
+        recipient: Recipient<NodeIndex>,
```

Comment:
You can use the AlephBFT `Recipient`, the special one is not necessary anymore.

Reply:
Done